Jenkinsfile_k8s

```diff
@@ -84,16 +81,6 @@ spec:
   }
 
   stages {
-    stage('Setup NPM and Yarn') {
-      steps {
-        sh '''
-          mkdir "${HOME}/.npm-packages"
-          npm config set prefix "${HOME}/.npm-packages"
-          npm install --global yarn
-        '''
-      }
-    }
-
     stage('NPM Install') {
```

Comment:
nit outdated from other PRs where it switched to yarn

Reply:
yea thats fair, it was kinda intentional. I felt NPM install was clearer than yarn, but I've switched it to "install dependencies"